### PR TITLE
Add support to aws_ec2_vpc_routetable

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject
To be merged after #911 
fixes: #913 
example:
```yaml
            region: us-east-1
            cidr_block: 15.0.0.0/16
            tags:
              module: aws_ec2_vpc_net
              this: alsoworks
            tenancy: dedicated
          - name: demo_vpc_subnet_r
            role: aws_ec2_vpc_subnet
            region: us-east-1
            vpc_name: demo_vpc_net_r
            cidr: 15.0.1.0/24
            tags:
              module: aws_ec2_vpc_subnet
              this: alsoworks
          - name: demo_routetable
            role: aws_ec2_vpc_routetable
            region: us-east-1
            vpc_name: demo_vpc_net_r
            tags:
              module: aws_ec2_vpc_routetable
              this: alsoworks
            subnets:
              - 'demo_vpc_subnet_r'
        credentials:
          filename: aws.yaml
          profile: default
```

### CODE REVIEW

File: config/Dockerfiles/tests.d/aws/01_aws_ec2_eip [hunk 1]
```diff
@@ -21,6 +21,3 @@ trap clean_up EXIT SIGHUP SIGINT SIGTERM
 pushd docs/source/examples/workspaces/${PROVIDER}
 
 linchpin -w . -v up "${TARGET}"
-
-# cleanup the network
-linchpin -w . -v destroy "${TARGET}"
```
Comment (14rcole) [inline config/Dockerfiles/tests.d/aws/01_aws_ec2_eip]: Why are we getting rid of the cleanup steps?

File: config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net [hunk 2]
```diff
@@ -18,7 +18,5 @@ trap clean_up EXIT SIGHUP SIGINT SIGTERM
 
 pushd docs/source/examples/workspaces/${PROVIDER}
 
-linchpin -w . -vvvv up "${TARGET}"
+linchpin -w . -v up "${TARGET}"
 
-# cleanup the network
-linchpin -w . -vvvv destroy "${TARGET}"
```
Comment (14rcole) [inline config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net]: Same as above

Comment (samvarankashyap) [reply config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net]: @14rcole : I observed that the cleanup is happening twice when i checked the logs that's why i removed the cleanup step.

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**